static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -0,0 +1,157 @@
+
+$accessibility-outline: #ffbf47;
+$correct-answer--background: #aedfb8;
+$user-response-was-right: #43b347;
+$user-response-was-wrong: #bb001a;
+
+.quiz {
+    margin: 0 0 $gs-baseline;
+    padding: 0;
```

Comment:
Yes it's not obvious. I think a comment would be helpful, but it may go stale, and I think we have too many overrides to make that a scalable solution. It goes back to that bigger problem, too much overriding and use of specificity.